### ISSUE TITLE
Intigrate cla-assistant.io

### ISSUE THREAD

**Issue by larsroettig** [extdn/extdn-phpcs#35]
The cla-assistant.io help assistant handle the legal side of contributions.

**Comment by fooman** [comment]
Good idea to bring this up early. ExtDN is no separate legal entity (at least so far) so not sure what the best approach here would be.

**Comment by schmengler** [comment]
At first I didn't believe it to be necessary because the code is not owned by a company. But since we might want to merge the rules into an official Magento ruleset in the future, I guess we are on the safe side if we integrate the CLA. @fooman ?
